forte_wrapper/spacy/spacy_processors.py

```diff
@@ -115,6 +172,40 @@ def _process_ner(self, result, input_pack):
                                    item.end_char)
             entity.ner_type = item.label_
 
+    def _process_umls_entity_linking(self, result, input_pack: DataPack):
+        """
+        Do UMLS medical entity linking with EntityLinker, and store medical
+        entity mentions and UMLS concepts.
+        Args:
```

Comment:
I think you need a blank space before `Args`. Please be careful with documentation styling. It is suggested to use automatic tools on this instead of manually typing them.